cla_backend/apps/legalaid/models.py

```diff
@@ -442,6 +443,9 @@ class Analytics:
     under_18_has_valuables = models.NullBooleanField(default=None)
     specific_benefits = JSONField(null=True, blank=True)
     disregards = JSONField(null=True, blank=True)
+    disregard_selection = models.CharField(
```

Comment:
```    disregard_selection = models.CharField(
        max_length=10, default=None, choices=DISREGARD_SELECTION, blank=True, null=True
    )```
  
  how comes this isn't like `choices=DISREGARD_SELECTION.CHOICES` like the other choice fields on this model(i.e `state = models.CharField(max_length=50, default=ELIGIBILITY_STATES.UNKNOWN, choices=ELIGIBILITY_STATES.CHOICES)`)

Reply:
This syntax works and is how this is implemented in the [Django 1.8 documentation](https://docs.djangoproject.com/en/1.8/ref/models/fields/). However, I have updated this to maintain consistency.